crates/primitives/starknet/src/starknet_serde/mod.rs

```diff
@@ -28,6 +28,49 @@ fn string_to_felt(hex_str: &str) -> Result<Felt252Wrapper, String> {
     }
 }
 
+#[allow(missing_docs)]
+#[derive(Error, Clone, Copy, Debug, scale_codec::Encode, scale_codec::Decode, scale_info::TypeInfo)]
+pub enum StarknetRpcApiError {
```

Comment:
Why put this struct in a file named `starknet_serde`?

Reply:
Should we create a new file just for this?

Reply:
wdyt?

Reply:
yup put it under an `errors` crate